### PR TITLE
add support for setting bot bandwidth

### PR DESCRIPTION
**Pull request by yunwah**
This PR is aimed at resolving #62 
Added support to the configuration file and arguments passed allowing users to specify the bots bandwidth usage. Prior to this PR I built it on docker and successfully was able to deploy it to the server. I have yet to test it with pure arguments.

### CODE REVIEW

**Comment by TerryGeng** [comment]
Thanks for your PR! LGTM.

**Comment by yunwah** [comment]
Glad this PR looks good! Quick question I had, is there any reason for choosing 200kb/s as the default bandwidth opposed to some other value?

**Comment by azlux** [comment]
>  is there any reason for choosing 200kb/s as the default bandwidth ?

No, it's here since the beginning. I don't thing It's a good value for music since default opus profile of pymumble is on quality mode now.

Mumble doc say : https://wiki.mumble.info/wiki/Murmur.ini#bandwidth default is 72000

The PR is good for me.

**Comment by yunwah** [comment]
> I don't thing It's a good value for music since default opus profile of pymumble is on quality mode now.
> Maybe we can make the default 96000 ?
I think this would be a good value as per your first reason. Another consideration area is the amount of bandwidth users hosting the bot may have. Setting the default would reduce overall bandwidth usage while still providing quality. If we're good on setting this as the value I'll update this PR with the new values.

File: configuration.default.ini [hunk 1]
```diff
@@ -32,6 +32,8 @@ comment = "Hi, I'm here to play radio, local music or youtube/soundcloud music.
 # default volume from 0 to 1.
 volume = 0.1
 stereo = True
+# bandwidth used by the bot
+bandwidth = 200000
```
Comment (azlux) [inline configuration.default.ini:36]: As discussed, can you set de default to 96000 ?

**Review by azlux — CHANGES_REQUESTED** [review]

File: configuration.default.ini (hunk 1 above)
Comment (yunwah) [reply configuration.default.ini:36]: Will do!

**Review by azlux — APPROVED** [review]

**Pull request merged**